sites/partners/pages/listings/[id]/applications/pending/index.tsx

```diff
@@ -198,6 +205,13 @@ const ApplicationsList = () => {
             </>
           )}
         </article>
+        {afsLastRun && (
+          <article className="flex max-w-screen-xl mx-auto pb-4 px-4 justify-end">
```

Comment:
Can this live within the article above so we don't have to duplicate sections? It seems functionally attached to the table vs a new section